scripts/neighbor_advertiser

```diff
@@ -384,10 +384,26 @@ def save_as_json(obj, file_path):
         json.dump(obj, outfile, sort_keys = True)
 
 
+def find_mirror_table_name():
```

Comment:
Can you add a comment for this function regarding why you need to do this look up? I think the reason is not clear and could be forgotten over time.